RiotSwiftUI/Modules/Room/Composer/View/Composer.swift

```diff
@@ -139,20 +131,39 @@ struct Composer: View {
                 .padding(.horizontal, horizontalPadding)
             }
             HStack(alignment: shouldFixRoundCorner ? .top : .center, spacing: 0) {
-                WysiwygComposerView(
-                    focused: $viewModel.focused,
-                    viewModel: wysiwygViewModel
-                )
-                .tintColor(theme.colors.accent)
-                .placeholder(viewModel.viewState.placeholder, color: theme.colors.tertiaryContent)
-                .onAppear {
-                    if wysiwygViewModel.isContentEmpty {
-                        wysiwygViewModel.setup()
+                // Use a GeometryReader to force the composer to fill the HStack
+                GeometryReader { _ in
+                    WysiwygComposerView(
+                        placeholder: viewModel.viewState.placeholder ?? "",
+                        viewModel: wysiwygViewModel,
+                        itemProviderHelper: nil,
+                        keyCommandHandler: handleKeyCommand,
+                        pasteHandler: nil
+                    )
+                    .clipped()
+                    .tint(theme.colors.accent)
+                    .focused($focused, equals: true)
```

Comment:
```suggestion
                    .focused($focused)
```
This should be equivalent